doc/execute-example.clj

```diff
@@ -1,9 +1,10 @@
 (ns app.funicular-example
```

Comment:
Not sure if the namespace and filename should be the same or is this on purpose?

Reply:
You can fix the namespace, it wasn't intentionally chosen.